infra/monitoring/bigquery/main.tf

```diff
@@ -0,0 +1,88 @@
+variable "region_name" {}
+variable "project_id" {}
+
+module "env" {
+  source = "../../common/env"
+}
+
+locals {
+  dataset_id                    = "${module.env.module_name}_metrics_${module.env.stage}"
+  standalone_durations_table_id = "${module.env.module_name}-standalone-engine-durations-${module.env.stage}"
+}
+
+provider "google" {
+  project = var.project_id
+  region  = var.region_name
+}
+
+resource "google_service_account" "bigquery_write" {
+  account_id   = "${module.env.module_name}-bigquery-write-${module.env.stage}"
+  display_name = "BigQuery Write"
+}
+
+resource "google_project_iam_member" "firestore_owner_binding" {
+  role   = "roles/bigquery.dataEditor"
+  member = "serviceAccount:${google_service_account.bigquery_write.email}"
+}
+
+resource "google_service_account_key" "bigquery_write_key" {
+  service_account_id = google_service_account.bigquery_write.name
+}
+
+resource "google_bigquery_dataset" "dataset" {
+  dataset_id                  = local.dataset_id
+  friendly_name               = "Lambdatization Metrics"
+  default_table_expiration_ms = 1576800000000 # 50 years ;-)
+  labels                      = module.env.default_tags
+}
+
+resource "google_bigquery_table" "standalone_engine_durations" {
+  dataset_id          = google_bigquery_dataset.dataset.dataset_id
+  table_id            = local.standalone_durations_table_id
+  labels              = module.env.default_tags
+  deletion_protection = false
+
+  time_partitioning {
+    type  = "DAY"
+    field = "timestamp"
+  }
+
+  schema = <<EOF
+[
+  {
+    "name": "engine",
+    "mode": "NULLABLE",
+    "type": "STRING"
+  },
+  {
+    "name": "initduration",
+    "mode": "NULLABLE",
+    "type": "INTEGER"
```

Comment:
❓ init duration is integer? could you specify also the unit

Reply:
This one is not populated for now, I'll remove it entirely